packages/fxa-admin-panel/server/config/index.ts

```diff
@@ -32,6 +32,33 @@ const conf = convict({
     // Note: This format is a number because the value needs to be in seconds
     format: Number,
   },
+  csp: {
+    enabled: {
+      default: true,
+      doc: 'Send "Content-Security-Policy" header',
+      env: 'CSP_ENABLED',
+    },
+    reportUri: {
+      default: 'https://accounts.firefox.com/_/csp-violation',
+      doc: 'Location of "report-uri" for full, blocking CSP rules',
+      env: 'CSP_REPORT_URI',
+    },
+    reportOnlyEnabled: {
+      default: false,
+      doc: 'Send "Content-Security-Policy-Report-Only" header',
+      env: 'CSP_REPORT_ONLY_ENABLED',
+    },
+    reportOnlyUri: {
+      default: 'https://accounts.firefox.com/_/csp-violation-report-only',
+      doc: 'Location of "report-uri" for report-only CSP rules',
+      env: 'CSP_REPORT_ONLY_URI',
+    },
+    extraImgSrc: {
```

Comment:
I believe this was added in payments to account for a CDN where we're storing some product images, but we shouldn't need it here since all the images will be hosted in the app.